packages/geo/src/lib/workspace/shared/feature-workspace.service.ts

```diff
@@ -81,11 +83,36 @@ export class FeatureWorkspaceService {
     const inMapExtentStrategy = new FeatureStoreInMapExtentStrategy({});
     const inMapResolutionStrategy = new FeatureStoreInMapResolutionStrategy({});
     const selectedRecordStrategy = new EntityStoreFilterSelectionStrategy({});
+    let styles = undefined;
```

Comment:
why not handle the selection style the same way as others?

getCommonVectorSelectedStyle or getCommonVectorStyle....

Reply:
Change made and get style from getCommonVector. Thanks for comment